web/src/utils/helpers.ts

```diff
@@ -158,3 +158,12 @@ export function getNetExchange(
 
   return netExchangeValue;
 }
+
+/**
+ * @returns {Boolean} true if agent is probably a mobile device.
+ */
+export function hasMobileUserAgent() {
+  return /android|blackberry|iemobile|ipad|iphone|ipod|opera mini|webos/i.test(
+    navigator.userAgent
+  );
+}
```

Comment:
```suggestion
export const hasMobileUserAgent = () => /android|blackberry|iemobile|ipad|iphone|ipod|opera mini|webos/i.test(
    navigator.userAgent
  );
```
Just a tiny bit smaller, I also think we can ditch iemobile and blackbarry. As far as I know they don't support the app anyhow 😅

And is not webOS only used in TVs these days?

Reply:
Well, it's also being used for the windlayer and may need support on mobile web, so I'm a little shy about removing blackberry and iemobile just yet

Maybe people are browsing our web app on their tvs? 😅